viewer/src/main/webapp/viewer-html/common/ScreenPopup.js

```diff
@@ -77,13 +78,13 @@ Ext.define ("viewer.components.ScreenPopup",{
             layout: 'fit',
             modal: false,
             renderTo: Ext.getBody(),
-            scrollable: true,
+            scrollable: this.config.details.usePopupScrolling,
             constrainHeader: true,
             iconCls: this.config.iconCls || "",
             bodyStyle: {},
-            cls: "screen-popup"
+            cls: "screen-popup",
```

Comment:
de slepende komma is een probleem voor de javascript compiler in 5.6.x, dus die zal ik oplossen